python/vowpalwabbit/pyvw.py

```diff
@@ -1278,22 +1278,19 @@ def __init__(self, action: int, score: float):
 
 
 class CCBSlotOutcome:
-    def __init__(
-        self, cost: Union["example", float] = 0, action_probs: List[ActionScore] = []
-    ):
-        if isinstance(cost, example):
-            self.from_example(cost)
-        else:
-            self.cost = cost
-            self.action_probs = action_probs
+    def __init__(self, cost: float = 0, action_probs: List[ActionScore] = []):
+        self.cost = cost
+        self.action_probs = action_probs
 
-    def from_example(self, ex: "example"):
-        self.cost = ex.get_ccb_cost()
-        self.action_probs = []
+    @staticmethod
+    def from_example(ex: "Example"):
+        cost = ex.get_ccb_cost()
```

Comment:
cost is optional based on the c++ code, so this won't match the types defined on line 1281